libs/MVS/SceneDensify.cpp

```diff
@@ -671,8 +671,10 @@ bool DepthMapsData::EstimateDepthMap(IIndex idxImage, int nGeometricIter)
 		#endif
 		if (prevDepthMapSize != size) {
 			BitMatrix mask;
-			if (OPTDENSE::nIgnoreMaskLabel >= 0 && DepthEstimator::ImportIgnoreMask(*depthData.GetView().pImageData, depthData.depthMap.size(), mask, (uint16_t)OPTDENSE::nIgnoreMaskLabel))
+			if (OPTDENSE::nIgnoreMaskLabel >= 0){
```

Comment:
are you sure this change is needed? it doesn't seem to do anything usefull

Reply:
If I don't call `ImportIgnoreMask`, if the image does not have a mask, `mask` does not get initialized. (Which might be the cause that triggers the crash/issue).

Reply:
Commands:

```
DensifyPointCloud /datasets/brighton_mask/opensfm/undistorted/openmvs/scene.mvs --resolution-level 2 --dense-config-file /datasets/brighton_mask/opensfm/undistorted/openmvs/Densify.ini --min-resolution 500 --max-resolution 2022 --max-threads 4 --number-views-fuse 2 --sub-resolution-levels 2 -w /datasets/brighton_mask/opensfm/undistorted/openmvs/depthmaps -v 0 --geometric-iters 0 --ignore-mask-label 0
(no crash, just incomplete point cloud)
```

![image](https://user-images.githubusercontent.com/1951843/175436297-721fb8f2-5344-4ddc-afae-4a470f4100af.png)


Followed by:

```
DensifyPointCloud --filter-point-cloud -1 -i /datasets/brighton_mask/opensfm/undistorted/openmvs/scene_dense.mvs -v 0 --geometric-iters 0 --ignore-mask-label 0
Segmentation fault (core dumped)
```

Point cloud with this patch:

![image](https://user-images.githubusercontent.com/1951843/175436413-0e3cf970-7f05-427b-9ded-9bc5b44c4dc4.png)


Thanks for looking into this.